Dockerfile.updater-core

```diff
@@ -181,10 +153,9 @@ ENV DEPENDABOT_HOME /home/dependabot
 COPY --chown=dependabot:dependabot LICENSE $DEPENDABOT_HOME
 COPY --chown=dependabot:dependabot updater $DEPENDABOT_HOME/dependabot-updater
 
-ENV PATH="$HOME/bin:/opt/go/bin:/opt/go_modules/bin:$PATH"
+ENV PATH="$HOME/bin:$PATH"
 ENV DEPENDABOT_NATIVE_HELPERS_PATH="/opt"
 
-USER dependabot
```

Comment:
Won't removing this line run CI as root?

Reply:
Oh CI doesn't run on this container at the moment? We might want to run common on this one, so perhaps we can do that as `dependabot`.